tox.ini

```diff
@@ -7,7 +7,7 @@ envlist =
 [gh-actions]
 python =
     3.7: py37
-    3.8: py38, docs, lint
```

Comment:
docs and lint are not run at the user side, only in CI, why is that?

Reply:
```text
(.venv) ❯❯ ~\..\PyGithub git:(pr/2463) tox
ROOT: tox-gh-actions won't override envlist because tox is not running in GitHub Actions
lint: commands[0]> pre-commit run --all-files --show-diff-on-failure
```

Reply:
/ping

Reply:
Never mind, you are right.